README.md

```diff
@@ -81,4 +81,4 @@ If you forget to sign your commits, the DCO bot will remind you and give you det
 
 ### License
 
-Copyright © [GNU General Public License v3.0](./LICENSE.md)
\ No newline at end of file
+Copyright © [GNU General Public License v3.0](./LICENSE.md)
```

Comment:
What's changed on this line?

Reply:
🤷🏽‍♂️

Reply:
Ryan's PR did this too: https://github.com/reactioncommerce/reaction/pull/5319/files#diff-04c6e90faac2675aa89e2176d2eec7d8L84 

Must be GitHub.com.